src-docs/src/views/drag_and_drop/drag_and_drop_disable_blocking.js

```diff
@@ -30,22 +34,35 @@ export default () => {
   return (
     <EuiDragDropContext onDragEnd={onDragEnd}>
       <EuiDroppable
-        droppableId="DROPPABLE_AREA"
+        droppableId="CUSTOM_HANDLE_DROPPABLE_AREA"
         spacing="m"
         withPanel
-        grow={false}
       >
         {list.map(({ content, id }, idx) => (
           <EuiDraggable
             spacing="m"
             key={id}
             index={idx}
             draggableId={id}
-            disableInteractiveElementBlocking
+            customDragHandle={true}
+            draggableContainer={true}
           >
-            <EuiButton fullWidth onClick={() => {}}>
-              {content}
-            </EuiButton>
+            {(provided) => (
+              <EuiPanel className="custom" paddingSize="m">
+                <EuiFlexGroup alignItems="center" gutterSize="l">
```

Comment:
Super visual minor spacing suggestions, I think this brings it back closer to what it looked like originally. Feel free to wait for @miukimiu on this one though
```suggestion
              <EuiPanel className="custom" paddingSize="s">
                <EuiFlexGroup alignItems="center" gutterSize="m">
```

Reply:
I added a new screenshot in the description to highlight this change.